EasyReflectometry/interfaces/refl1d/interface.py

```diff
@@ -7,14 +7,15 @@
 import numpy as np
 from easyCore.Objects.Inferface import ItemContainer
 
-from EasyReflectometry.calculators.refl1d import Refl1d as Refl1d_calc
 from EasyReflectometry.experiment.model import Model
-from EasyReflectometry.interfaces.interfaceTemplate import InterfaceTemplate
 from EasyReflectometry.sample.item import MultiLayer
 from EasyReflectometry.sample.layer import Layer
 from EasyReflectometry.sample.material import Material
 from EasyReflectometry.sample.material import MaterialMixture
 
+from ..interfaceTemplate import InterfaceTemplate
```

Comment:
We should probably standardize on absolute imports, unless specifically required.
PEP8 says something pretty strict `Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports.` but we can define certain use cases for it, if necessary.

Reply:
The reason why I think it is alright here is that the interfaceTemplate is the "base" class for all the contained interfaces (calculators).  This "base" defines the (abstract) methods that all children (the different calculators) must implement.

What we obtain is that we then ensures that all children of the given base at least can be called with the methods specified in the base.  For the calculators (abstract methods in InterfaceTemplate) that seems to be:
- `reset_storage`
- `fit_func`
- `sld_profile`
But also the following that are used in `model.py`
- `create`
- `assign_material_to_layer`
- `add_layer_to_item`
- `remove_layer_from_item`
- `add_item_to_model`
- `remove_item_from_model`

I plan to introduce these changes in a coming PR where I also rename `interface` to `calculator`